src/librbd/crypto/openssl/DataCryptor.cc

```diff
@@ -46,7 +46,7 @@ int DataCryptor::init(const char* cipher_name, const unsigned char* key,
 
 DataCryptor::~DataCryptor() {
   if (m_key != nullptr) {
-    explicit_bzero(m_key, EVP_CIPHER_key_length(m_cipher));
+    memset(m_key, 0, EVP_CIPHER_key_length(m_cipher));
```

Comment:
```suggestion
    memset_s(m_key, 0, EVP_CIPHER_key_length(m_cipher));
```

Reply:
It should be part of the C11 standard but it's missing I'd vote for a wrapper around `memset_s` vs the less standard `explicit_bzero`